sources/sqlserver/README.md

```diff
@@ -0,0 +1,68 @@
+# HarbourBridge: SQLServer-to-Spanner Evaluation and Migration
+
+HarbourBridge is a stand-alone open source tool for Cloud Spanner evaluation,
+using data from an existing PostgreSQL,MySQL,SQL server database. This README provides
+details of the tool's SQLserver capabilities. For general HarbourBridge information
```

Comment:
Please also mention the alias in the first para

Reply:
Sure, changes will be available in the next PR update